device_attributes/index.bs

```diff
@@ -0,0 +1,192 @@
+<pre class='metadata'>
+Title: Device Attributes API
+Shortname: device-attributes
+Level: 1
+Editor: Anqing Zhao 130190, Google LLC https://www.google.com, anqing@google.com
```

Comment:
Find this number at https://www.w3.org/users/myprofile, according to https://tabatkins.github.io/bikeshed/#metadata-editor.

Reply:
Registered and changed the account number here. Thanks for the guide.